src/content/1.7/development/coding-standards/_index.md

```diff
@@ -124,4 +140,6 @@ All PrestaShop files MUST start with the PrestaShop license block:
  * @license   https://opensource.org/licenses/AFL-3.0  Academic Free License (AFL 3.0)
  * International Registered Trademark & Property of PrestaShop SA
  */
-``` 
+```
+
+[compile-assets]: {{< ref "/1.7/development/compile-assets/_index.md" >}}
```

Comment:
Note that you can't use footnotes links inside notices, so if you accept my previous suggestion you will have to delete this.

```suggestion
```